swap_meet/item.py

```diff
@@ -1,2 +1,53 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0, age = 0):
+        # If no id is provided, generate a unique id using the uuid package 
+        # Instead of using uuid package to generate unique ids for Item instances,
+        #  I could use the built-in id function to generate unique ids based on the object's memory address. 
+        # This can simplify the code and avoid the dependency on an external package.
+        if id is None:
+            self.id = uuid.uuid4().int  # could be if use built-in function self.id = id(self) 
+        elif not isinstance(id, int):
+            raise ValueError("id must be an integer")
+        # Otherwise, use the provided id
+        else:
+            self.id = id
+        # Check that condition is within valid range (0-5)
+        if not 0 <= condition <= 5:
```

Comment:
Careful about statements like this. They sometimes have unexpected behavior. Prefer writing out the logical expression:

```python
if not 0 <= condition or not condition <= 5:
```

or easier to read:

```python
if condition < 0 or condition > 5:
```